porting/finalize/index.rst

```diff
@@ -3,15 +3,23 @@
 Finalizing the port
 ===================
 
-Once most frequently used features are working on your port you can start thinking about finalizing it, i.e. building recovery and preparing the installer so that it is easier for people to install and test it. This can give valuable feedback that will potentially help you to find and diagnose remaining issues faster than you could have done on your own.
+Once most frequently used features are working on your port you can start thinking about finalizing it, i.e. building recovery and preparing the installer so that it is easier for people to install and test it.
+This can give valuable feedback that will potentially help you to find and diagnose remaining issues faster than you could have done on your own.
 
-Previously, your port has had the rootfs and system image coexisting on the userdata partition. These need to be moved to the system partition in order to ensure a maximum of available space for user data. This is done by modifying the code, building ``recovery.img`` and subsequently rebuilding the boot image.
+Previously, your port has had the rootfs and system image coexisting on the userdata partition.
+These need to be moved to the system partition in order to ensure a maximum of available space for user data.
+This is done by modifying the code, building ``recovery.img`` and subsequently rebuilding the boot image.
 
-With the necessary components prepared, these should first be installed in their correct places manually for testing purposes. Once it has been confirmed that the port boots and works as it should, the final step is to prepare and test an installer config file that will permit the automation of the whole process.
+With the necessary components prepared, these should first be installed in their correct places manually for testing purposes.
+Once it has been confirmed that the port boots and works as it should, the final step is to prepare and test an installer config file that will permit the automation of the whole process.
+
+.. note::
```

Comment:
I would suggest to drop this note. Or at least don't format it as a note. It stands out and interrupts the reading flow, but I don't think this is needed here